ctapipe/tools/process.py

```diff
@@ -238,6 +245,7 @@ def start(self):
         self.log.info("(re)compute DL2: %s", self.should_compute_dl2)
         self.event_source.subarray.info(printer=self.log.info)
 
+        event = None
```

Comment:
Why this line? Everything else inside this function is inside the for-loop, where `event` gets overwritten

Reply:
It is for the case that the source has no triggered events that `event` is not undefined.

Reply:
Ah, sorry, it's actually not needed in this PR, comes from other code I was also adding

Reply:
Ok then, can stay here I think